cypress/e2e/app-spec.cy.js

```diff
@@ -0,0 +1,47 @@
+const tvmazeSearchStub = JSON.stringify({
+  "id": 82,
+  "name": "Game of Thrones",
+  "officialSite": "http://www.hbo.com/game-of-thrones",
+  "image": {
+    "medium": "https://static.tvmaze.com/uploads/images/medium_portrait/190/476117.jpg",
+    "original": "https://static.tvmaze.com/uploads/images/original_untouched/190/476117.jpg"
+  }
+})
+
+const ourStub = JSON.stringify({
+  "prescription": {
+    "id": "1",
+    "show_id": "82",
+    "message": "wow"
+  }
+})
+
```

Comment:
What do you think about abstracting this out into it's own file, like the tvmaze data?

Reply:
Can this file be renamed to reflect the user flow?

Reply:
Updated filename!